.circleci/README.md

```diff
@@ -66,7 +66,7 @@ Every fly application needs a `fly.toml` file to manage deployment. You can find
  flyctl apps create
 ```
 
-The `flyctl apps create` command creates an interactive session which requests for things like the 'App name' and 'Organization name'. The App name is expected to be a unique name. You can type in a unique App name, or leave it blank if you want a unique Fly auto-generated name to avoid namespace collisions. Fly will also reserve the app name on the platform, for when you deploy.
+The `flyctl apps create` command creates an interactive session that requests things like the 'App name' and 'Organization name'. You can type a unique App name, or leave it blank if you want a Fly to auto-generate name to avoid namespace collisions. Fly will also reserve the app name on the platform, for when you deploy.
```

Comment:
`flyctl init` starts an interactive session that ...

Also asks for builder and internal port. (not mentioned)